source/common/common/BUILD

```diff
@@ -348,7 +348,11 @@ genrule(
 genrule(
     name = "generate_version_linkstamp",
     outs = ["lib/version_linkstamp.h"],
-    cmd = "$(location :generate_version_linkstamp.sh) >> $@",
+    cmd = select({
+        # Only iOS builds typically follow this logic, OS/X is built as a normal binary
+        "//bazel:apple": "$(location :generate_version_linkstamp.sh) Library >> $@",
```

Comment:
@junr03 to double check

Reply:
Good morning. Please note that this PR blocks adoption of RBE of //test/... on windows, and is becoming more of a showstopper for us at 2 weeks stale.

So for background, we cautiously adopted a neutral behavior for this patch for iOS mobile, this PR is a no-op to the existing behavior on that platform. (Library) remains (Library) at this time on apple (non-10.x) targets.

https://github.com/envoyproxy/envoy/pull/11291#issuecomment-632827626 is simply a discussion topic to consider if we would want to make this consistent on iOS mobile, but this does not need to be changed at this time via this PR.